gluecodium/src/test/resources/smoke/comments/input/Comments.lime

```diff
@@ -78,9 +78,10 @@ class comments {
     // @return nicely documented
     fun returnCommentOnly(undocumented: String): String
 
-    // Some very useful property.
+    // Status of some property.
     // @get Gets some very useful property.
```

Comment:
Doesn't the lime files will have a @return value for Java @gets ?

Reply:
Outdated due to new changes.